src/main/resources/openapi/cases.yaml

```diff
@@ -183,6 +183,34 @@ paths:
               schema:
                 $ref: './problem.yaml'
 
+  /cases/{case_id}/transcripts:
+    get:
+      tags:
+        - Cases
+      summary: Allows the retrieval of transcripts for a given case_id
+      description: |-
+        Retrieves all transcripts for a case_id.
+      parameters:
+        - in: path
+          name: case_id
+          schema:
+            type: integer
+          description: "case_id is the internal cas_id of the case."
+          required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/transcripts'
+        '500':
```

Comment:
Can you add in the 404 response for a non-existent case id

Reply:
definition added